.gitignore

```diff
@@ -17,7 +17,8 @@
 .glide/
 
 # Build Binaries
-bin
+/bin
+/hack/tools/bin
```

Comment:
Any reason to add the `/` in front of `bin` and `hack`?
It would be cleaner if these lines are consistent with the rest of the file where `/` is not added to the beginning.